docs/DeveloperGuide.adoc

```diff
@@ -881,26 +881,31 @@ image::DoneSessionSequenceDiagram.png[]
 
 1. The `LogicManager` uses the `TaTrackerParser` to first parse the user command.
 
-2. The `TaTrackerParser` sees that the command is a type *sessions* and passes the command
+2. The `TaTrackerParser` sees that the command is a *sessions* type and passes the command
 to the `SessionCommandParser`.
 
-3. The `SessionCommandParser` sees that the command is a type of *done* and passes the
+3. The `SessionCommandParser` sees that the command is a *done* type and passes the
 arguments to the `DoneSessionCommandParser`.
 
 4. The `DoneSessionCommandParser` creates a `DoneSessionCommand` with the given index.
 
 5. `LogicManager` calls `DoneSessionCommand#execute()` method.
 
-6. The `DoneSessionCommand`checks whether the current session called by the user has a recurring period.
+6. The `DoneSessionCommand`
+checks whether the current session called by the user has a recurring period.
 
-.. If it doesn't, ...
-.. If it does, a new session with the updated date will be added to `Model#UniqueSessionList()`.
+    a. If it does, a new session with the updated date will be added to `Model#UniqueSessionList()`.
+    b. If it does not has a recurring period, it will move on to *Step 6*.
 
 7. The current session will be removed from `Model#UniqueSessionList`.
 
-8. The updated filtered session list will be displayed to the user.
+8. The updated session list will be displayed to the user.
 
-The `session add`, `session edit` and `session delete` commands have been implemented in a similar manner.
+
+==== Implementation of Session Add, Edit and Delete
+
+The `session add`, `session edit` and `session delete` commands have been implemented in a similar manner
```

Comment:
hmm, i think `session edit` and `session delete` are similar.

could you include a short paragraph for `session add`? can refer to aakanksha's group add desciription. Cause in this case, its different since it needs to check that the module exists.